we pass -D_FORTIFY_SOURCE=3 in clang by default, so every TU generates a warning
since this passes 2 on top.
make it pass 3 instead like we do
--
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1974,7 +1974,7 @@
       # Non-chromium code is not guaranteed to compile cleanly with
       # _FORTIFY_SOURCE. Also, fortified build may fail when optimizations are
       # disabled, so only do that for Release build.
-      fortify_level = "2"
+      fortify_level = "3"
 
       # ChromeOS's toolchain supports a high-quality _FORTIFY_SOURCE=3
       # implementation with a few custom glibc patches. Use that if it's
